app/views/admin/conference_domains/show.haml

```diff
@@ -0,0 +1,95 @@
+.row
+  .col-md-12
+    %h1
+      Custom domain
+    %p.text-muted
+      Point your own domain to #{@conference.title}
+%hr
+.row
+  .col-md-12
+    %table.table
+      %thead
+        %th Domain Name
+        %th
+          Correctly points to #{@conference.short_title}
+          %a{ title: 'Click here to know more about this feature!', href: 'http://github.com/openSUSE/osem/wiki' }
+            %span.glyphicon.glyphicon-question-sign
+        %th Actions
+      %tbody
+        %tr
+          %td
+            = @conference.custom_domain
+          %td
+            - if ENV['OSEM_HOSTNAME'].present?
+              - ck_domain = ConferenceDomainsService.new(@conference).check_custom_domain
+              - if ck_domain
+                %span.glyphicon.glyphicon-ok
+              - else
+                %span.glyphicon.glyphicon-remove
+            - else
+              %i
+                feature disabled
+          %td
+            .btn-group
+              = link_to 'Edit', edit_admin_conference_domain_path(@conference.short_title),
+                method: :get, class: 'btn btn-primary'
+              = link_to 'Delete', admin_conference_domain_path(@conference.short_title),
+                method: :delete, class: 'btn btn-danger'
+.row
+  .col-md-9.text-right
+    = link_to '#status-help', class: 'btn btn-default',  "data-toggle"=>"collapse" do
+      Need Help?
+.row
+  .col-md-12
+    .collapse#status-help
```

Comment:
I'm not sure if this should be collapsed.
When you set a domain for the conference, you get a flash notice saying "now follow step 3", but "step 3" is not visible anywhere, and you have to *guess* that it's under the "Need help?" link.

Reply:
Sounds good, I don't think there is any harm in keeping it visible at all times.

Reply:
ok!